pos_meal_voucher/static/src/xml/pos_meal_voucher.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-extend="Orderline">
+        <t t-jquery=".product-name" t-operation="prepend">
+            <t t-if="line.get_product().meal_voucher_ok === true &amp;&amp;
+                widget.pos.config.meal_voucher_display_product_screen">
+                <i class="fa fa-lg fa-cutlery"/>
+            </t>
+        </t>
+    </t>
+
+    <t t-extend="OrderWidget">
+        <t t-jquery="div.line" t-operation="append">
+            <div class="meal-voucher">
+                (<span>Meal Voucher: </span> <span class="value">0.00 €</span>)
```

Comment:
EURO symbol to remove ? Currency is not always euro ;)

Reply:
Just a placeholder. will not be used.

See Odoo code : https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/static/src/xml/pos.xml#L1321